dojo/tools/netsparker/parser.py

```diff
@@ -60,21 +65,20 @@ def get_findings(self, filename, test):
                 cwe=cwe,
                 static_finding=True,
             )
-
-            if item["State"].find("FalsePositive") != -1:
+            state = item.get("State", None)
+            if state == "FalsePositive":
                 finding.active = False
                 finding.verified = False
                 finding.false_p = True
                 finding.mitigated = None
                 finding.is_mitigated = False
-
-            if item["State"].find("AcceptedRisk") != -1:
+            elif state == "AcceptedRisk":
                 finding.risk_accepted = True
 
             if (
                 (item["Classification"] is not None)
-                and (item["Classification"]["Cvss"] is not None)
-                and (item["Classification"]["Cvss"]["Vector"] is not None)
+                and (item["Classification"].get("Cvss") is not None)
+                and (item["Classification"].get("Cvss").get("Vector") is not None)
```

Comment:
There is possibly a bit more "compressed version"
```python
if not item.get("Classification", {}).get("Cvss", {}).get("Vector"):
```

Reply:
https://github.com/DefectDojo/django-DefectDojo/blob/c182e9ca9d8f981c15de2018f948fe69c4d1a800/unittests/scans/netsparker/netsparker_many_findings.json#L4224 --> AttributeError: 'NoneType' object has no attribute 'get'

Reply:
Ou. I'm sorry, you are right

Reply:
No problem, I understand that you did not go through all unittest files. Good idea from your side.